platform/viewer/public/config/newlantern.js

```diff
@@ -0,0 +1,129 @@
+const config = {
+  routerBasename: '/',
+  extensions: [],
+  modes: [],
+  showStudyList: false,
+  dataSources: [
+    {
+      friendlyName: 'New Lantern DICOM server',
+      namespace: 'org.ohif.default.dataSourcesModule.dicomweb',
+      sourceName: 'dicomweb',
+      configuration: {
+        name: 'New Lantern',
+        wadoUriRoot:
+          'https://healthcare.googleapis.com/v1/projects/new-lantern/locations/us-central1/datasets/development/dicomStores/wen-dev/dicomWeb',
+        qidoRoot:
+          'https://healthcare.googleapis.com/v1/projects/new-lantern/locations/us-central1/datasets/development/dicomStores/wen-dev/dicomWeb',
+        wadoRoot:
+          'https://healthcare.googleapis.com/v1/projects/new-lantern/locations/us-central1/datasets/development/dicomStores/wen-dev/dicomWeb',
```

Comment:
Needs to be set during runtime, maybe based on environment variable?

Reply:
The urls are also set on an instance level. Maybe we could retrieve one instance and get the base url. 

https://app.oldlantern.dev/admin/core/instance/